tests/unit/accounts/test_services.py

```diff
@@ -464,19 +470,29 @@ def test_disable_password(self, user_service, reason, expected):
         [(True, None), (True, DisableReason.CompromisedPassword), (False, None)],
     )
     def test_is_disabled(self, user_service, disabled, reason):
+        request = pretend.stub(
+            remote_addr="127.0.0.1",
+            ip_address=IpAddressFactory.create(ip_address="127.0.0.1"),
```

Comment:
If we never assert the value of the created IP, allow the factory to generate a random one?

```suggestion
            ip_address=IpAddressFactory.create(),
```

Similar note applies to other cases in this module.

Reply:
3e752256f